src/fb/ReactDOMFiberFBEntry.js

```diff
@@ -14,20 +14,29 @@ var ReactDOMFiber = require('ReactDOMFiberEntry');
 Object.assign(
   ReactDOMFiber.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED,
   {
-    // These are real internal dependencies that are trickier to remove:
-    ReactBrowserEventEmitter: require('ReactBrowserEventEmitter'),
     ReactErrorUtils: require('ReactErrorUtils'),
+    // used by react-fb/ReactDOM on FB www
+    DOMProperty: require('DOMProperty'),
     ReactFiberErrorLogger: require('ReactFiberErrorLogger'),
-    ReactFiberTreeReflection: require('ReactFiberTreeReflection'),
-    ReactDOMComponentTree: require('ReactDOMComponentTree'),
+    // ReactInstanceMap are used to track instances
+    // for reactComponentExpect and ReactLayeredComponentMixin_DEPRECATED
     ReactInstanceMap: require('ReactInstanceMap'),
-    // This is used for ajaxify on www:
```

Comment:
This comment explains *why* we use it, can we keep it? It got lost with the move of this line.